### ISSUE TITLE
Configure Key Vault secrets in production Azure

### ISSUE THREAD

**Issue by thekaveman** [cal-itp/benefits#1699]
This task is to create entries for _all_ new app settings / environment settings that will be deployed with the Veterans feature, while ensuring that the only active pathways remain MST Senior/Login.gov and MST Courtesy Cards.

We're just going to bite the bullet and configure it all now to get each environment in sync.

### Veterans general

```
veteran-auth-provider-client-name
veteran-auth-provider-scope
veteran-auth-provider-claim
veteran-auth-provider-scheme
```

### MST Veterans

```
mst-veteran-group-id
mst-veteran-verifier-name
mst-veteran-verifier-active = False
```

### MST Senior/Login.gov

Add the following:

```
senior-auth-provider-client-name
senior-auth-provider-scope
senior-auth-provider-claim
senior-auth-provider-scheme
mst-senior-verifier-name
mst-senior-verifier-active = True
```

Remove the following:

```
auth-provider-client-name
auth-provider-scope
auth-provider-claim
mst-oauth-verifier-name
```

### MST Courtesy Cards

Add the following:

```
courtesy-card-verifier-name
courtesy-card-verifier-active = True
```

Remove the following:

```
courtesy-card-verifier
```

## Updates for in-progress agencies

These agencies are in various stages of progress towards launch, and should remain fully **`inactive`** with no flows.

### SBMTD

Add the following:

```
sbmtd-senior-group-id
sbmtd-payment-processor-client-cert
sbmtd-payment-processor-client-cert-private-key
sbmtd-payment-processor-client-cert-root-ca
sbmtd-senior-verifier-name
sbmtd-senior-verifier-active = False
sbmtd-payment-processor-name
sbmtd-payment-processor-api-base-url
sbmtd-payment-processor-api-access-token-endpoint
sbmtd-payment-processor-api-access-token-request-key
sbmtd-payment-processor-api-access-token-request-val
sbmtd-payment-processor-card-tokenize-url
sbmtd-payment-processor-card-tokenize-func
sbmtd-payment-processor-card-tokenize-env
sbmtd-agency-short-name
sbmtd-agency-long-name
sbmtd-agency-merchant-id
sbmtd-agency-active = False
sbmtd-agency-jws-signing-alg
```

### SacRT

Add the following:

```
sacrt-senior-verifier-name
sacrt-senior-verifier-active = False
```

Remove the following:

```
sacrt-oauth-verifier-name
```

**Comment by angela-tran** [comment]
I've finished cleaning up and checking over the secrets for `dev` and `test`.

I'm now working on `prod` secrets. One thing to note is we need to create a group for MST Veterans in production Littlepay before we can set `mst-veteran-group-id`.

Should I go ahead and create that, or does someone else want to do it?

**Comment by thekaveman** [comment]
> One thing to note is we need to create a group for MST Veterans in production Littlepay before we can set mst-veteran-group-id.

Good note, that will be covered in #1485. You can just use a placeholder for now since we still want to create the secret.

**Comment by angela-tran** [comment]
I've finished adding new secrets for `prod`. "New secrets" includes brand-new secrets, like ones for veterans and sbmtd-senior, as well as renamed secrets (oauth-verifier -> senior-verifier).

I haven't removed secrets for `prod` because the code currently deployed in production Benefits is still using those.

Here are the ones to remove once we've done a new release to prod:
- [ ] auth-provider-client-name
- [ ] auth-provider-scope
- [ ] auth-provider-claim
- [ ] mst-oauth-verifier-name
- [ ] courtesy-card-verifier
- [ ] sacrt-oauth-verifier-name

**Comment by thekaveman** [comment]
Nice work @angela-tran! I created a new issue to track those removals after we deploy. Good to close this one?

**Comment by angela-tran** [comment]
> Nice work @angela-tran! I created a new issue to track those removals after we deploy. Good to close this one?

Yep, I'll close this one out. Thanks!

**Comment by angela-tran** [comment]
I meant to also note that these secrets are ones I set to placeholder values for the time being:
- [x] mst-veteran-group-id - placeholder value
- [x] sbmtd-senior-group-id - placeholder value
- [x] sbmtd-payment-processor-client-cert - placeholder value
- [x] sbmtd-payment-processor-client-cert-private-key - placeholder value
- [x] sbmtd-payment-processor-client-cert-root-ca - placeholder value
